tools/sizes/code_functions.csv

```diff
@@ -327,11 +327,11 @@ asm/non_matchings/code/z_actor/func_800B9098.s,func_800B9098,0x800B9098,0x5
 asm/non_matchings/code/z_actor/func_800B90AC.s,func_800B90AC,0x800B90AC,0x12
 asm/non_matchings/code/z_actor/func_800B90F4.s,func_800B90F4,0x800B90F4,0xB
 asm/non_matchings/code/z_actor/func_800B9120.s,func_800B9120,0x800B9120,0x14
-asm/non_matchings/code/z_actor/Actor_Init.s,Actor_Init,0x800B9170,0x71
+asm/non_matchings/code/z_actor/func_800b9170.s,func_800b9170,0x800B9170,0x71
```

Comment:
Er.. so its not init? Can I get a capital :b:?

Reply:
No, the function was wrongly named